schemas/rum/_common-schema.json

```diff
@@ -63,6 +63,12 @@
           "description": "Whether this session is currently active. Set to false to manually stop a session",
           "default": true,
           "readOnly": true
+        },
+        "start_reason": {
+          "type": "string",
+          "description": "The precondition that led to the creation of the session",
+          "enum": ["app_start", "15m_expire", "4h_expire", "stop_api", "background_event"],
```

Comment:
I would say it is better to use something like `inactivity_timeout` and `max_duration`, because essentially these constants may change at some point